app/models/concerns/encrypt_value.rb

```diff
@@ -19,7 +19,7 @@ def is_encryptable?(str)
   end
 
   def is_decryptable?(str)
```

Comment:
How about we turn the entire method into this while we're at it
```ruby
!encryption_key.blank? && matches_prefix(str)
```

Reply:
Oh nice. Perhaps `is_encryptable` could follow the same pattern.

Reply:
gotta love De Morgan's Laws :)